ingestion_tools/scripts/importers/visualization_config.py

```diff
@@ -174,7 +177,11 @@ def _create_config(self, alignment_metadata_path: str) -> dict[str, Any]:
         volume_info = tomogram.get_output_volume_info()
         voxel_size = round(volume_info.voxel_size, 3)
         resolution = (voxel_size * 1e-10,) * 3
-        layers = [self._to_tomogram_layer(tomogram, volume_info, resolution)]
+        t = time()
+        print("Start contrast limit computation for", tomogram)
```

Comment:
Does this need to stay in the code? I assume it was here for performance tuning.

Reply:
@Bento007 Thanks for the feedback! Actually, we added some feedbacks as the contrast limit can take a little bit of time depending on the method, or the level of the pyramid that is used as input for the computation. The idea was to not have just the scripts that hangs for few seconds without any information about why it is. If that's something that you feel is not necessary, we can remove it.

Reply:
No, that's fine. It would be worth adding a comment as to why we add this metric, since we aren't doing it anywhere else.

Reply:
I updated the PR with the comment :)